### PR TITLE
Adding new messages

### PR DESCRIPTION
**Pull request by iblancasa**
- Parsing some messages from the initialiazation of Connext DDS
- Parsing other messages related to:
  - Skipped interfaces
  - Inconsistent transport/discovery configuration

### CODE REVIEW

File: src/initialization/logs.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDS_DISCOVERY_PEERS not found"])
+    regex.append([initialization.on_no_file_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:file NDDS_DISCOVERY_PEERS not found"])
+    regex.append([debug.on_ignored_message,
+                  r"DDS_DomainParticipantFactory_initializeI:Welcome to NDDS"])
+    regex.append([debug.on_ignored_message,
+                  r"DDS_DiscoveryQosPolicy_get_default:no environment variable or file NDDS_DISCOVERY_PEERS"])
+    regex.append([initialization.on_no_ndds_home,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDSHOME not found"])
+    regex.append([initialization.on_no_environment_ndds_qos_profiles,
```
Comment (pleonex) [inline src/initialization/logs.py]: Check if the `RTIOsapi_envVarOrFileGet` expression are parsed in the config module.

File: src/initialization/logs.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    """Return the regular expressions and functions list for this module."""
+    regex = []
+
+    # NDDS_DISCOVERY_PEERS
+
+    regex.append([initialization.on_no_environment_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDS_DISCOVERY_PEERS not found"])
+    regex.append([initialization.on_no_file_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:file NDDS_DISCOVERY_PEERS not found"])
+    regex.append([debug.on_ignored_message,
```
Comment (pleonex) [inline src/initialization/logs.py]: Move the ignore messages to the debug module.

File: src/events/logs.py [hunk 3]
```diff
@@ -78,6 +83,8 @@ def get_regex_list():
     regex.append([events.on_fail_delete_flowcontrollers,
                   r"PRESParticipant_destroyOneFlowControllerWithCursor:" +
                   r"has (\d+) writers on flowcontroller"])
+    regex.append([events.on_inconsistent_transport_discovery_configuration,
+                  r"\[D0000\|ENABLE\]DDS_DomainParticipant_enableI:Automatic participant index failed to initialize\. PLEASE VERIFY CONSISTENT TRANSPORT \/ DISCOVERY CONFIGURATION\."])
```
Comment (pleonex) [inline src/events/logs.py]: Remove the context `[D0000|ENABLE]` from the regular expression. They could not appear in some situations.

File: src/events/events.py [hunk 4]
```diff
@@ -47,6 +47,9 @@ def on_get_valid_interface(match, state):
         log_cfg("Valid interface: %s (%s multicast)" % (match[1], multicast),
                 state)
 
+def on_skipped_interface(match, state):
+    log_event("Skipped interface: %s" % match[0],state,2)
```
Comment (pleonex) [inline src/events/events.py]: Missing spaces in the arguments. Try running pylama on the new code.

File: src/initialization/logs.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                  r"DDS_DomainParticipantFactory_initializeI:Welcome to NDDS"])
+    regex.append([debug.on_ignored_message,
+                  r"DDS_DiscoveryQosPolicy_get_default:no environment variable or file NDDS_DISCOVERY_PEERS"])
+    regex.append([initialization.on_no_ndds_home,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDSHOME not found"])
+    regex.append([initialization.on_no_environment_ndds_qos_profiles,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDS_QOS_PROFILES not found"])
+
+
```
Comment (pleonex) [inline src/initialization/logs.py]: Too many blank lines

File: src/initialization/logs.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Log Parser for RTI Connext.
```
Comment (pleonex) [inline src/initialization/logs.py]: I would say to add these logs into the config module. They are _similar_ and at the moment there isn't enough logs to split into a new module.

File: src/logger.py [hunk 7]
```diff
@@ -129,6 +129,14 @@ def log_error(text, state, level=0):
         log(text, level, state, COLORS['FAIL'])
 
 
+def log_init(text, state, level=2):
```
Comment (pleonex) [inline src/logger.py]: Merge this function with `log_cfg`, setting the default verbosity to 2 by default.

**Review by pleonex — CHANGES_REQUESTED** [review]

**Comment by iblancasa** [comment]
I think that all is OK now

File: src/events/logs.py [hunk 8]
```diff
@@ -37,6 +37,9 @@ def get_regex_list():
                   r"NDDS_Transport_UDPv4_InterfaceListener_onInterface:" +
                   r"interface ([\d\.]+) \((\w+)\), enabled=(\d), " +
                   r"multicast=(\d)"])
+    regex.append([events.on_skipped_interface,
+                  r"\[D0000\|ENABLE\]NDDS_Transport_UDPv4_query_interfaces:" +
```
Comment (pleonex) [inline src/events/logs.py]: Remove the context log `[D000|ENABLE]` from the regex.

File: src/events/logs.py [hunk 9]
```diff
@@ -161,4 +168,16 @@ def get_regex_list():
     regex.append([events.on_participant_initial_peers,
                   r'DDS_DomainParticipantDiscovery_enableI:value of: ' +
                   r'initial_peers="(.+)"'])
+    regex.append([events.on_no_environment_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:environment variable " +
```
Comment (pleonex) [inline src/events/logs.py]: I recommend you to make these expressions general by using a capture group for the env var or file names. Like  `RTIOsapi_envVarOrFileGet: file (\w+) not found"`

**Review by pleonex — CHANGES_REQUESTED** [review]

**Comment by iblancasa** [comment]
I think this is de last :+1: I squashed all the commits into one... because there are some dummy commits

**Comment by iblancasa** [comment]
And... now?

**Review by pleonex — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**